recipe/conda_build_config.yaml

```diff
@@ -1,2 +1,5 @@
 MACOSX_SDK_VERSION:        # [osx and x86_64]
   - "10.12"                # [osx and x86_64]
```

Comment:
```suggestion
```
@hmaarrfk, can we remove this? I believe the baseline SDK version is now 10.13 because the linter says so.

Reply:
yes, can be removed.